lib/core/src/Data/Quantity.hs

```diff
@@ -154,6 +156,9 @@ instance FromText Percentage where
         err = TextDecodingError
             "expected a value between 0 and 100 with a '%' suffix (e.g. '14%')"
 
+instance Buildable Percentage where
+    build (Percentage p) = mempty <> build p <> "%"
```

Comment:
`mempty` is good when one wants to align stuff vertically. Here it doesn't really buy us much. 

```suggestion
build (Percentage p) = build p <> "%"
```

or

```suggestion
build (Percentage p) = mempty
    <> build p
    <> "%"
```